Feature/products

Grzybkowa opened the pull request:
This pull request introduces products section. 
[
](https://trello.com/c/lkDOnDCq/17-sekcja-products-na-tablet-desktop)
[
](https://trello.com/c/mEGfKCLM/6-sekcja-products)

Grzybkowa committed:
added products section

- added styles for products section
- added base for products section

Grzybkowa committed:
added products section

- added styles for products section
- added base for products section

Grzybkowa committed:
new class added to _products.scss

Grzybkowa committed:
new class added to _products.scss

DamianPiechota committed:
footer mobile added

benbrzezinski committed:
Merge branch 'main' into feature/footer

Grzybkowa committed:
products img added to imgaes folder and products.html

Grzybkowa committed:
products img added to imgaes folder and products.html

Grzybkowa committed:
rename products images

Grzybkowa committed:
rename products images

Grzybkowa committed:
products section responsive on mobile and tablet

Grzybkowa committed:
products section responsive on mobile and tablet

Grzybkowa committed:
changes in _products.scss

Grzybkowa committed:
changes in _products.scss

Grzybkowa committed:
_products.scss - media screen for desktop added

Grzybkowa committed:
_products.scss - media screen for desktop added

benbrzezinski committed:
Fixed bug with focus on buttons in header and deleted filter drop shadow on mobile menu.

Grzybkowa committed:
fixed scss classes

Grzybkowa committed:
fixed scss classes

Grzybkowa committed:
removed redundant comment

Grzybkowa committed:
removed redundant comment

barteknr1 committed:
Merge pull request #13 from barteknr1/bugfix/buttons-header

Fixed bug with focus on buttons in header and deleted filter drop sha…

barteknr1 commented on src/partials/products.html:
```diff
@@ -0,0 +1,142 @@
+<section class="container products">
```
Poproszę zrobić to tak:
-section class="section" - id daj na section zamiast 100%. Nazwa id to Products
-pod spodem div class="container"

bp998 committed:
advantages section

bp998 committed:
adding rwd

ApolinarySzumski committed:
feat: adding favicon and title

Dodany został favicon wersja 6 oraz tytuł

DamianPiechota committed:
update mobile footer

DamianPiechota committed:
Merge branch 'main' into feature/footer

Grzybkowa committed:
chnages in products.html

figure and picture parts has been removed from code

Grzybkowa committed:
chnages in products.html

figure and picture parts has been removed from code

Grzybkowa committed:
new classes added to _products.scss

- :: after added ;
- background-image added to ::after

Grzybkowa committed:
new classes added to _products.scss

- :: after added ;
- background-image added to ::after

DamianPiechota committed:
ID added

DamianPiechota committed:
Merge branch 'feature/footer' of https://github.com/barteknr1/Projekt-grupowy into feature/footer

barteknr1 committed:
Merge pull request #16 from barteknr1/feature/favicon

feat: adding favicon and title

barteknr1 commented on src/partials/products.html:
```diff
@@ -0,0 +1,55 @@
+<section class="container products-block">
```
Brakuje div class="container products" pod section. Klasę products-block na razie usuń

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,323 @@
+
+  border-radius: 24px;
+  align-items: center;
+  justify-content: center;
```
justify-content nic nie zmienia

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,323 @@
+
+.product__description {
+  padding: 30px 39px;
+  font-family: DM Sans;
```
DM Sans jest w body, nie trzeba pisać

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,323 @@
+}
+
+.product__name {
+  font-family: Titan One;
```
Brakuje grupy do jakiej należy Titan One. Dodam później do tego zmienną

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,323 @@
+  background-color: var(--color-white);
+  fill: var(--color-header);
+  justify-content: center;
+  align-items: center;
```
justify-content i align-items nie mają wpływu na zawartość więc można to usunąć

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,323 @@
+    content: '';
+    position: absolute;
+    top: -120px;
+    right: 15px;
```
Wystarczy samo left

barteknr1 reviewed (changes requested):
Coś jest nie halo z obrazkiem Ice Coffee na tablecie, zerknij sobie

bp998 committed:
advantages bugfix

barteknr1 committed:
Merge pull request #15 from barteknr1/feature/advantages

advantages section

bp998 committed:
gallery setup

DamianPiechota committed:
update

bp998 committed:
adding gallery

barteknr1 committed:
Update _gallery.scss

barteknr1 committed:
Merge pull request #18 from barteknr1/bugfix/gallery

Bugfix/gallery

benbrzezinski committed:
Created header and menu on tablet.

benbrzezinski committed:
Replace import.

Grzybkowa committed:
code review

- ice-coffee img width changed /is comply with Figma/;
- class="section" added;
- <div class="container section"> added ;
- class="products-block" removed;
- justify-content removed from .product__background;
- font-family removed from .product__description;
- generic-family added to font-family:Titan-One;
-justify-content and align-items removed from .product__arrow-right;
-right: 15px removed from.product;

Grzybkowa committed:
code review

- ice-coffee img width changed /is comply with Figma/;
- class="section" added;
- <div class="container section"> added ;
- class="products-block" removed;
- justify-content removed from .product__background;
- font-family removed from .product__description;
- generic-family added to font-family:Titan-One;
-justify-content and align-items removed from .product__arrow-right;
-right: 15px removed from.product;

barteknr1 committed:
Update _footer.scss

barteknr1 committed:
Merge pull request #11 from barteknr1/feature/footer

footer mobile added

benbrzezinski committed:
Merge branch 'main' into feature/header-tablet

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -1,15 +1,15 @@
   margin-bottom: 150px;
 }
 
 .products__top-line {
```
Nazwij to __label

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,315 @@
+  letter-spacing: 0.04em;
+  text-align: center;
+  margin-top: 161px;
+  margin-bottom: 30px;
```
text-align nie ma wpływu na zawartość

barteknr1 commented on src/sass/components/_products.scss:
```diff
@@ -0,0 +1,315 @@
+
+.product__description {
+  padding: 30px 39px;
+  font-size: 14px;
```
14px jest w body, można usunąć

barteknr1 commented on src/partials/products.html:
```diff
@@ -0,0 +1,57 @@
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+          ut labore et. dolore magna aliqua facilisis.
+        </p>
+        <svg class="product__arrow-right">
+          <use href="./images/icons.svg#icon-arrow-right"></use>
+        </svg>
```
Wsadź to w button, to przycisk z funkcją

benbrzezinski committed:
Changed font size in slogan title on mobile.

Grzybkowa committed:
second code review

- products__titles renamed (products__label);
- text-align removed from .product__name;
- font-size removed from .product__description;
- <button> and .product__button added;
- @media added directly to class selectors;

Grzybkowa committed:
second code review

- products__titles renamed (products__label);
- text-align removed from .product__name;
- font-size removed from .product__description;
- <button> and .product__button added;
- @media added directly to class selectors;

barteknr1 committed:
Merge pull request #20 from barteknr1/feature/header-tablet

Created header and menu on tablet.

ApolinarySzumski committed:
fix/bugfixing in customersreview

barteknr1 committed:
Merge pull request #21 from barteknr1/bugfix/customersreview

fix/bugfixing in customersreview

Grzybkowa committed:
bugfix line 27 (letter-spacing)

Grzybkowa committed:
Merge branch 'feature/products' of https://github.com/barteknr1/Projekt-grupowy into feature/products
